server/channels/store/sqlstore/channel_store.go

```diff
@@ -1181,7 +1206,7 @@ func (s SqlChannelStore) getAllChannelsQuery(opts store.ChannelSearchOpts, forCo
 	if forCount {
 		selectStr = "count(c.Id)"
 	} else {
-		selectStr = "c.*, Teams.DisplayName AS TeamDisplayName, Teams.Name AS TeamName, Teams.UpdateAt AS TeamUpdateAt"
+		selectStr = strings.Join(channelSliceColumns("c"), ", ") + ", Teams.DisplayName AS TeamDisplayName, Teams.Name AS TeamName, Teams.UpdateAt AS TeamUpdateAt"
```

Comment:
There's a few other cases where we concatenate these on, but they're passed into the query builder, so they seem safe to me, but I could use a second opinion from @isacikgoz on that

Reply:
Done, used query builder now.